.circleci/config.yml

```
@@ -643,10 +643,28 @@ jobs:
                 -DPIKA_WITH_SPINLOCK_DEADLOCK_DETECTION=On
           working_directory: /home/circleci/project/build
       - run:
-          name: Build pika Arm64
+          name: Building pika Arm64
           command: |
             ninja -k 0 pika
           working_directory: /home/circleci/project/build
+      - run:
+          name: Building tests Arm64
+          command: |
+            ninja -k 0 tests
+          working_directory: /home/circleci/project/build
+      - run:
+          name: Running tests Arm64
+          when: always
+          command: |
+              ulimit -c unlimited
+              ctest \
+                -j8 \
+                --timeout 120 \
```

```suggestion
                --timeout 300 \
```
and/or
`-j4`. I think oversubscribing the machine this much is a bit excessive and makes some tests run a lot longer.